### DISCUSSION TITLE
About wallabag 2.7.0

### DISCUSSION THREAD

**Discussion by nicosomb** [wallabag/wallabag#6884] [Announcements]
Last year, @j0k3r opened ["Upcoming wallabag version 3.0.0 & 2.6.0"](https://github.com/wallabag/wallabag/discussions/6177). 

🚀 Since this discussion, we released 2.6.0 (and patches). These releases contains fixes and lots of technical stuff to go forward with our stack (Symfony 4.4). Thank you @yguedidi. 

About 3.0.0, nothing has changed. You can read again what we decided here https://github.com/wallabag/wallabag/discussions/6177. 

Because wallabag 3.0.0 will be released when it will be ready (in wallabag team, there is no pressure and no releases calendar), we added a new milestone: 2.7.0. 

This release will contain lots of new features. You can [have a look here](https://github.com/wallabag/wallabag/milestone/79?closed=1) to see those already implemented. 

👉 To avoid upgrade issues (as we had with 2.6.0), **we'll publish beta releases in coming weeks**. 
I hope that the great wallabag community will help us to test this new version 😊 

We also try to update a large part of the translations: I will publish an announcement on weblate (https://hosted.weblate.org/projects/wallabag/). If you want to fix or improve your language in wallabag, don't hesitate to join us 🫶

We accept contributions (fixes and new features, translations, websites configuration, and so on) so contact us if you have ideas ❤️

As I said before, no pressure in our team. But I would like to publish wallabag 2.7.0 before the end of the year 🤞

**Comment by bcutter** [comment]
> But I would like to publish wallabag 2.7.0 before the end of the year

Which one? :-)

I want to help translate stuff, currently running 2.6.13 quite an ammount of texts is not localized. Where to start and will it help for current releases or only starting 2.7.0?

**Reply by nicosomb** [reply]
Hello @bcutter 👋 

> Which one? :-)

We are always too optimistic 😅 

If you want to help us by translating some text, you can have a look here: https://hosted.weblate.org/projects/wallabag/
If you work on `2.6` branch, your translations will be available on 2.6.x releases and more (2.7, etc). 
If you work on `master` branch, they will be available only for 2.7 and more.

**Reply by skn** [reply]
Do we have an ETA for 2.7? I ask because I created PR on `master` branch https://github.com/wallabag/wallabag/pull/8339 which I do not mind porting to `2.6` branch as well (once the current one gets merged). Let the know @nicosomb

**Reply by skn** [reply]
oh well, went ahead and did it anyway https://github.com/wallabag/wallabag/pull/8346
